example/spec/helpers/granuleUtils.js

```diff
@@ -208,7 +208,18 @@ const waitForGranuleRecordsInList = async (stackName, granuleIds, additionalQuer
 
 const waitForGranuleRecordUpdatedInList = async (stackName, granule, additionalQueryParams = {}) => await pWaitFor(
   async () => {
-    const fieldsIgnored = ['timestamp', 'updatedAt'];
+    // Ignore the fields generated by CMR. The date string will be in a
+    // different ISO format.
+    const fieldsIgnored = [
+      'beginningDateTime',
+      'endingDateTime',
+      'error',
+      'files', // FUTURE: files contain a "type" field in elasticsearch that does not exist in Postgres
+      'lastUpdateDateTime',
+      'productionDateTime',
+      'updatedAt',
+    ];
```

Comment:
So. We'd talked about parsing these timestamps and continuing to compare. That turns out to be a little messy because we have at least one that doesn't parse correctly coming back from CMR. This is the format that's currently being produced:

`lastUpdateDateTime: '2018-04-25T21:45:45.524053',`

That looks to be expected as that example came straight from `test-cmr-utils.js`, which has not been modified. 

There's also an issue with `pWaitFor` where changing values on the `results` or `granule` object doesn't work every time. That's is no doubt fixable but I'm thinking it's not worth it. I added a unit test to check that the date values are the same and this helper isn't really interested in that comparison. It's just a byproduct of it trying to find the granule.

DEFINITELY feel free to disagree as I realize this is partially just wanting to get it out but I do tend to think that this helper is already doing more than it should. Plus there are new tests that cover it. 

I added a topic in the next arch meeting. Maybe we can discuss there?

Reply:
I think this method should just await list granule and if the match behavior is expected should either throw or have `false` fail the test, or ....